core/server/src/main/java/alluxio/master/file/options/DeleteFileOptions.java

```diff
@@ -0,0 +1,109 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.master.file.options;
+
+import alluxio.thrift.DeleteTOptions;
+
+import com.google.common.base.Objects;
+
+import javax.annotation.concurrent.NotThreadSafe;
+
+/**
+ * Method options for deleting a file or a directory.
+ */
+@NotThreadSafe
+public final class DeleteFileOptions {
```

Comment:
If use the DeleteOptions as the alluxio's delete option, the filesystemMaster will have alluxio.underfs.options.DeleteOptions and alluxio.master.file.options.DeleteOptions, So, use the full package name to identify the two DeleteOptions?